zebra-consensus/Cargo.toml

```diff
@@ -15,7 +15,7 @@ displaydoc = "0.2.1"
 jubjub = "0.6.0"
 lazy_static = "1.4.0"
 once_cell = "1.8"
-pairing = "0.19.0"
+pairing = "0.20.0"
```

Comment:
`pairing` is not used by anything in `zebra-consensus` anymore!

```suggestion
```

Reply:
> `pairing` is not used by anything in `zebra-consensus` anymore!

We'll need to update the lock file as well, so it might be better to create another PR.